templates/repo/release_tag_header.tmpl

```diff
@@ -10,10 +10,12 @@
 					<a class="{{if .PageIsTagList}}active {{end}}item" href="{{.RepoLink}}/tags">{{ctx.Locale.PrettyNumber .NumTags}} {{ctx.Locale.TrN .NumTags "repo.tag" "repo.tags"}}</a>
 				{{end}}
 			</h2>
-			{{if .EnableFeed}}
-				<a class="rss-icon gt-mx-3" href="{{.RepoLink}}/{{if .PageIsTagList}}tags{{else}}releases{{end}}.rss" data-tooltip-content="{{ctx.Locale.Tr "rss_feed"}}">{{svg "octicon-rss" 18}}</a>
-			{{end}}
 		</div>
+		{{if .EnableFeed}}
+			<a class="ui small button" href="{{.RepoLink}}/{{if .PageIsTagList}}tags{{else}}releases{{end}}.rss">
+				{{svg "octicon-rss" 18}} {{ctx.Locale.Tr "rss_feed"}}
```

Comment:
Why is it suddenly static text and no longer a tooltip?

Reply:
> A special exception was made for the Releases pages (which contains
> text, not a tooltip), where an RSS feed would be particularly
> beneficial to users.

In other words, it is assumed that the person / primary user of that page who will be seeing the button will probably not bother enough to look at the tooltip. If it's a blocker, I can remove it.

It is also a method to provide some sort of a visual distinction from the other, repo-wide icon.